### DISCUSSION TITLE
old value

### DISCUSSION THREAD

**Discussion by jebau** [MrFibunacci/laravel-form-components#26] [Q&A]
Hello, is it possible to use old value submited ? like this ?
{!! isset($customer) ? $customer->firstname : old('firstname') !!}

thanks for your help

**Comment by Okipa** [comment]
Hi @jebau, the old value is automatically retrieved when one is defined for the name you did set.

If no old value is defined, the value will be set from the bound item.

However, if you need to set a custom value that would override the one that is automatically retrieved, you can set it from the ´value´ attribute.

I don't know if this is answering your question, please let me know if you need more explanation.

You may give more details about your use case, it could help me to understand what you are trying to achieve.
